app/src/ui/branches/pull-request-badge.tsx

```diff
@@ -14,7 +15,10 @@ interface IPullRequestBadgeProps {
   /** The GitHub repository to use when looking up commit status. */
   readonly repository: GitHubRepository
 
-  readonly onBadgeRef?: (ref: HTMLDivElement | null) => void
+  /** Whether or not the check runs popover is open */
+  readonly showCIStatusPopover?: boolean
```

Comment:
This attribute is optional because this component is also used in our pull request quick view component and it is not used as a button there. Since the pull request quick view component is feature flagged to development and work is currently stopped on it, I did not think it was worth the time handling the "non-button" view of this component. This can be reworked if that work is picked up again.